Only in ca-certificates: .deps.mk
diff -uri ca-certificates.old/Makefile ca-certificates/Makefile
--- ca-certificates.old/Makefile	2014-03-13 13:43:00.000000000 +0100
+++ ca-certificates/Makefile	2014-05-19 16:55:57.508048517 +0200
@@ -2,7 +2,7 @@
 # Makefile
 #
 
-CERTSDIR = /usr/share/ca-certificates
+CERTSDIR = /share/ca-certificates
 SUBDIRS = mozilla spi-inc.org
 
 all:
@@ -17,7 +17,7 @@
 
 install:
 	for dir in $(SUBDIRS); do \
-	  mkdir $(DESTDIR)/$(CERTSDIR)/$$dir; \
+	  mkdir -p $(DESTDIR)/$(CERTSDIR)/$$dir; \
 	  $(MAKE) -C $$dir install CERTSDIR=$(DESTDIR)/$(CERTSDIR)/$$dir; \
 	done
 	for dir in sbin; do \
Only in ca-certificates/mozilla: certdata2pem.c
diff -uri ca-certificates.old/mozilla/Makefile ca-certificates/mozilla/Makefile
--- ca-certificates.old/mozilla/Makefile	2013-09-07 09:40:28.000000000 +0200
+++ ca-certificates/mozilla/Makefile	2014-05-19 16:55:57.509048517 +0200
@@ -1,14 +1,10 @@
-#
-# Makefile
-#
-
-all:
-	python certdata2pem.py
-
+all: a.out
+	./a.out
+a.out:
+	$(HOSTCC) -static -Wall -Wextra -O0 -g certdata2pem.c
 clean:
 	-rm -f *.crt
-
 install:
-	for p in *.crt; do \
-	 install -m 644 $$p $(CERTSDIR)/$$p ; \
+	for p in *.crt; do\
+		install -D -m 644 "$$p" "$(CERTSDIR)/$$p";\
 	done
diff -uri ca-certificates.old/sbin/Makefile ca-certificates/sbin/Makefile
--- ca-certificates.old/sbin/Makefile	2013-09-07 09:40:28.000000000 +0200
+++ ca-certificates/sbin/Makefile	2014-05-19 16:55:57.507048517 +0200
@@ -8,4 +8,5 @@
 clean:
 
 install:
-	install -m755 update-ca-certificates $(DESTDIR)/usr/sbin/
+	install -m 755 -d $(DESTDIR)/sbin/
+	install -m 755 update-ca-certificates $(DESTDIR)/sbin/
diff -uri ca-certificates.old/sbin/update-ca-certificates ca-certificates/sbin/update-ca-certificates
--- ca-certificates.old/sbin/update-ca-certificates	2014-03-13 13:43:00.000000000 +0100
+++ ca-certificates/sbin/update-ca-certificates	2014-05-19 16:55:57.506048517 +0200
@@ -38,8 +38,8 @@
 done
 
 CERTSCONF=/etc/ca-certificates.conf
-CERTSDIR=/usr/share/ca-certificates
-LOCALCERTSDIR=/usr/local/share/ca-certificates
+CERTSDIR=/share/ca-certificates
+LOCALCERTSDIR=/local/share/ca-certificates
 CERTBUNDLE=ca-certificates.crt
 ETCCERTSDIR=/etc/ssl/certs
 
diff -uri ca-certificates.old/spi-inc.org/Makefile ca-certificates/spi-inc.org/Makefile
--- ca-certificates.old/spi-inc.org/Makefile	2013-09-07 09:40:28.000000000 +0200
+++ ca-certificates/spi-inc.org/Makefile	2014-05-19 16:55:57.506048517 +0200
@@ -8,6 +8,6 @@
 
 install:
 	for p in *.crt; do \
-	 install -m 644 $$p $(CERTSDIR)/$$p ; \
+	 install -D -m 644 $$p $(CERTSDIR)/$$p ; \
 	done
 
Only in ca-certificates: .targs.mk
