--- ca-certificates.org/Makefile
+++ ca-certificates/Makefile
@@ -2,7 +2,7 @@
 # Makefile
 #
 
-CERTSDIR = /usr/share/ca-certificates
+CERTSDIR = /share/ca-certificates
 SUBDIRS = mozilla spi-inc.org
 
 all:
diff -ur ca-certificates-20120212.orig//mozilla/Makefile ca-certificates-20120212/mozilla/Makefile
--- ca-certificates-20120212.orig//mozilla/Makefile	2011-11-06 19:23:24.000000000 -0600
+++ ca-certificates-20120212/mozilla/Makefile	2012-04-11 02:45:30.691561468 -0500
@@ -10,5 +10,5 @@
 
 install:
 	for p in *.crt; do \
-	 install -m 644 $$p $(CERTSDIR)/$$p ; \
+	 install -D -m 644 $$p $(CERTSDIR)/$$p ; \
 	done
diff -ur ca-certificates-20120212.orig//sbin/Makefile ca-certificates-20120212/sbin/Makefile
--- ca-certificates-20120212.orig//sbin/Makefile	2007-02-02 00:23:19.000000000 -0600
+++ ca-certificates-20120212/sbin/Makefile	2012-04-11 02:53:36.755557626 -0500
@@ -8,4 +8,5 @@
 clean:
 
 install:
-	install -m755 update-ca-certificates $(DESTDIR)/usr/sbin/
+	mkdir -p $(DESTDIR)/bin
+	install -m755 update-ca-certificates $(DESTDIR)/bin/
diff -ur ca-certificates-20120212.orig//sbin/update-ca-certificates ca-certificates-20120212/sbin/update-ca-certificates
--- ca-certificates-20120212.orig//sbin/update-ca-certificates	2011-11-06 19:23:24.000000000 -0600
+++ ca-certificates-20120212/sbin/update-ca-certificates	2012-04-11 02:35:01.190560840 -0500
@@ -38,8 +38,8 @@
 done
 
 CERTSCONF=/etc/ca-certificates.conf
-CERTSDIR=/usr/share/ca-certificates
-LOCALCERTSDIR=/usr/local/share/ca-certificates
+CERTSDIR=/share/ca-certificates
+LOCALCERTSDIR=/local/share/ca-certificates
 CERTBUNDLE=ca-certificates.crt
 ETCCERTSDIR=/etc/ssl/certs
 
diff -ur ca-certificates-20120212.orig//spi-inc.org/Makefile ca-certificates-20120212/spi-inc.org/Makefile
--- ca-certificates-20120212.orig//spi-inc.org/Makefile	2007-02-02 00:23:19.000000000 -0600
+++ ca-certificates-20120212/spi-inc.org/Makefile	2012-04-11 02:44:35.435558255 -0500
@@ -8,6 +8,6 @@
 
 install:
 	for p in *.crt; do \
-	 install -m 644 $$p $(CERTSDIR)/$$p ; \
+	 install -D -m 644 $$p $(CERTSDIR)/$$p ; \
 	done
 
