--- libressl-2.0.0.org/crypto/compat/issetugid_linux.c
+++ libressl-2.0.0/crypto/compat/issetugid_linux.c
@@ -4,7 +4,9 @@
  */
 
 #include <errno.h>
+#ifdef __GLIBC__
 #include <gnu/libc-version.h>
+#endif
 #include <string.h>
 #include <sys/types.h>
 #include <unistd.h>
@@ -33,15 +35,19 @@
 	 * info: http://lwn.net/Articles/519085/
 	 *
 	 */
+#ifdef __GLIBC__
 	const char *glcv = gnu_get_libc_version();
 	if (strverscmp(glcv, "2.19") >= 0) {
+#endif
 		errno = 0;
 		if (getauxval(AT_SECURE) == 0) {
 			if (errno != ENOENT) {
 				return 0;
 			}
 		}
+#ifdef __GLIBC__
 	}
+#endif
 #endif
 	return 1;
 }
--- libressl-2.0.0.org/crypto/compat/getentropy_linux.c
+++ libressl-2.0.0/crypto/compat/getentropy_linux.c
@@ -24,7 +24,6 @@
 #include <sys/ioctl.h>
 #include <sys/resource.h>
 #include <sys/syscall.h>
-#include <sys/sysctl.h>
 #include <sys/statvfs.h>
 #include <sys/socket.h>
 #include <sys/mount.h>
