From 3ebbf1c0dd5dfaee56e6fae28cefb7756180ff22 Mon Sep 17 00:00:00 2001
From: Hiltjo Posthuma <hiltjo@codemadness.org>
Date: Sat, 15 Feb 2014 15:09:55 +0100
Subject: [PATCH 2/3] remove unused code

Signed-off-by: Hiltjo Posthuma <hiltjo@codemadness.org>
---
 quark.c | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/quark.c b/quark.c
index 152476a..565e091 100644
--- a/quark.c
+++ b/quark.c
@@ -31,27 +31,11 @@ typedef struct {
 	const char *mimetype;
 } MimeType;
 
-typedef struct _Param Param;
-struct _Param {
-	const char *key;
-	const char *value;
-	Param *next;
-};
-
 typedef struct {
 	int type;
 	int fd;
-	const char *hostname;
-	const char *resource;
-	Param *params;
 } Request;
 
-typedef struct {
-	const char *hostname;
-	const char *resource;
-	void (*handle)(const Request *r);
-} RequestHandler;
-
 static const char HttpOk[]           = "200 OK";
 static const char HttpMoved[]        = "301 Moved Permanently";
 static const char HttpUnauthorized[] = "401 Unauthorized";
-- 
1.8.5.3

